### PR TITLE
CNS-631-add-ibc-modules

### PR DESCRIPTION
**Pull request by Yaroms**

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -556,6 +572,7 @@ func New(
 		// TODO: Check alternatives to this
 		// AddRoute(distrtypes.RouterKey, distr.NewCommunityPoolSpendProposalHandler(app.DistrKeeper)).
 		AddRoute(upgradetypes.RouterKey, upgrade.NewSoftwareUpgradeProposalHandler(&app.UpgradeKeeper)).
+		// AddRoute(ibcexported.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper))
```
Comment (oren-lava) [inline app/app.go:575]: should it be in a comment? these lines didn't exist before

File: scripts/init_chain.sh [hunk 2]
```diff
@@ -11,7 +11,8 @@ if ! command_exists jq; then
 fi
 
 rm -rf ~/.lava
-lavad init validator --chain-id lava
+chainID="lava-local-1"
```
Comment (oren-lava) [inline scripts/init_chain.sh:14]: why? what was wrong with the original chain-id "lava"?

File: scripts/relayer/init_chain_2.sh [hunk 3]
```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```
Comment (oren-lava) [inline scripts/relayer/init_chain_2.sh:1]: I'm guessing this is used to start another node in parallel to a running node. Can you put a small comment here to the script usage for future reference?

File: scripts/relayer/init_chain_2.sh [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        | jq '.app_state.gov.params.voting_period = "4s"' \
+        | jq '.app_state.gov.params.expedited_voting_period = "3s"' \
+        | jq '.app_state.gov.params.expedited_min_deposit[0].denom = "ulava"' \
+        | jq '.app_state.gov.params.expedited_min_deposit[0].amount = "200"' \
+        | jq '.app_state.gov.params.expedited_threshold = "0.67"' \
+        | jq '.app_state.mint.params.mint_denom = "ulava"' \
+        | jq '.app_state.staking.params.bond_denom = "ulava"' \
+        | jq '.app_state.crisis.constant_fee.denom = "ulava"' \
+        | jq '.app_state.epochstorage.params.epochsToSave = "5"' \
+        | jq '.app_state.epochstorage.params.epochBlocks = "4"' \
```
Comment (oren-lava) [inline scripts/relayer/init_chain_2.sh:42]: add downtime_duration=1s

File: scripts/relayer/init_relay.sh [hunk 5]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (oren-lava) [inline scripts/relayer/init_relay.sh:1]: can you add a comment about what this script does?

**Review by oren-lava — CHANGES_REQUESTED** [review]

File: app/app.go (hunk 1 above)
Comment (Yaroms) [reply app/app.go:575]: yes, currently we dont enable this, maybe in the future

File: scripts/init_chain.sh (hunk 2 above)
Comment (Yaroms) [reply scripts/init_chain.sh:14]: omer want to differentiate it from mainnet chain id so there wont be mix ups

File: scripts/relayer/init_chain_2.sh (hunk 4 above)
Comment (Yaroms) [reply scripts/relayer/init_chain_2.sh:42]: done

File: scripts/relayer/init_chain_2.sh (hunk 3 above)
Comment (Yaroms) [reply scripts/relayer/init_chain_2.sh:1]: done

File: scripts/init_chain.sh (hunk 2 above)
Comment (omerlavanet) [inline scripts/init_chain.sh]: we can do lava since mainnet isn't lava-1

**Pull request merged**